lib/protocol/protocol.js

```diff
@@ -210,13 +210,16 @@ function routeConfiguringFunction (driver) {
     throw new Error('Drivers used with MJSONWP must implement `executeCommand` or `execute`');
   }
 
-  // return a function which will add all the routes to the driver
-  return function addRoutes (app, basePath = DEFAULT_BASE_PATH) {
+  // return a function which will add all the routes to the driver. Here extraMethods might be
+  // passed in as defined by Appium plugins, so we need to add those to the default list
+  return function addRoutes (app, basePath = DEFAULT_BASE_PATH, extraMethods) {
```

Comment:
I would rather make everything after `app` argument as options, so the signature would look like `addRoutes(app, ops = {}) `

Reply:
👍